.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -10,6 +10,10 @@ name: Week 1 Step 3
 Week 1 Step 3 ⬤⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 10-15 minutes
 
 ## Getting Cat Pics with Postman 🐱
+### ✅  Task:
+
+- [ ]  With Postman, send a GET request to the Cat Picture API and receive a cat picture with "Serverless" written on it 
+- [ ]  Comment the picture you receive in this Issue/Pull Request to Move On
```

Comment:
Clarify this task:
`To move on, comment the picture you receive from the API in this issue/pull request.`